mipengine/node/monetdb_interface/common_actions.py

```diff
@@ -23,7 +23,7 @@ def create_table_name(
     """
     if table_type not in {"table", "view", "merge"}:
         raise TypeError(f"Table type is not acceptable: {table_type} .")
-    if node_id not in {"global", config.node.identifier}:
+    if node_id not in {"global", config.identifier}:
```

Comment:
Again, why does the database interface need to know about the types of tables we have defined in a higher level of the system? Table types "table","view","merge" are artifacts we have defined on a higher level of the system. They are defined as part of the node context. The db interface should not be coupled to that. These checks should be performed before the request to create a table reaches the db interface.